HW_3_2_6.py

```diff
@@ -0,0 +1,10 @@
+li = []
```

Comment:
At the start of your programm try to add the comments:
#########################
#Author:
#Date:
#Task:
#
#########################